files/metricspusher.sh

```
@@ -28,13 +28,20 @@ while true; do
     echo "Error: Failed to fetch metrics from Hoprd API"
   fi
 
-  if ! hopr_safe_allowance=$(curl -s --max-time 10 -H 'accept: application/json' -H "X-Auth-Token: ${HOPRD_API_TOKEN}" "http://hoprd:3001/api/v3/account/balances" | jq -r '.safeHoprAllowance'); then
+  if ! account_balance=$(curl -s --max-time 10 -H 'accept: application/json' -H "X-Auth-Token: ${HOPRD_API_TOKEN}" "http://hoprd:3001/api/v3/account/balances"); then
     echo "Error: Failed to fetch Hopr node balances"
   fi
+  safe_allowance=$(echo $account_balance | jq -r '.safeHoprAllowance')
+  safe_balance=$(echo $account_balance | jq -r '.safeHopr')
+
+  # Append the new metric entry
+  metrics+="\nhopr_safe_allowance ${safe_allowance}"
```

Shellcheck (0.10.0)

[warning] In POSIX sh, += is undefined.

(SC3024)